docs/docs/migration/v0.0.56-RC6-to-v0.0.57-0.md

```diff
@@ -0,0 +1,145 @@
+### Migrate from Magda v0.0.56-RC6 to v0.0.57-0
```

Comment:
I just realised that I had one more comment that I didn't put in before - can we call this just `0.0.56 to 0.0.57`, without the extra versions? :)

Reply:
The file name is changed & all relevant keywords have also been changed in the document